nixos/tests/initrd-network-ssh/default.nix

```diff
@@ -37,12 +35,16 @@ import ../make-test-python.nix ({ lib, ... }:
       {
         environment.etc = {
           knownHosts = {
-            text = concatStrings [
+            text = lib.concatStrings [
               "server,"
-              "${toString (head (splitString " " (
-                toString (elemAt (splitString "\n" config.networking.extraHosts) 2)
-              )))} "
-              "${readFile ./ssh_host_ed25519_key.pub}"
+              "${toString
+                (lib.head
+                  (lib.splitString " "
+                    (toString
+                      (lib.elemAt
+                        (lib.splitString "\n" config.networking.extraHosts)
+                        2))))} "
+              "${lib.readFile ./ssh_host_ed25519_key.pub}"
```

Comment:
I would revert the formatting of this specific hunk as it is worse afterwards.

Reply:
Ok, I'll fix this later.